test/utils.js

```diff
@@ -62,5 +62,24 @@ module.exports = {
 		});
 	},
 	web3: web3,
-	TestRPC: TestRPC
+	TestRPC: TestRPC,
+	promisifyContractFactory: function(contractFactory) {
+		contractFactory.newAsync = function() {
+			var args = arguments;
```

Comment:
It is necessary, otherwise, inside `function(resolve, reject)`, `arguments are set to `{ '0': [Function resolve], '1': [Function reject] }`.